### ISSUE TITLE
centerness-loss

### ISSUE THREAD

**Issue by Eaphan** [Stick-To/Object-Detection-API-Tensorflow#16]
center_pos_loss = -center_gt*tf.log_sigmoid(center_pred) * loc
        center_neg_loss = -(1.-center_gt)*(-center_pred+tf.log_sigmoid(center_pred)) * (1.-loc)
        center_loss = tf.reduce_sum(center_pos_loss) + tf.reduce_sum(center_neg_loss)

The location not in groundtruth boxes should not be inclued while computing centerness-loss, and the code of binary sigmod entropy may be wrong.
-(y_true*log(y_pred) + (1 - y_truth) log(1 - y_pred))

**Comment by Stick-To** [comment]
I think the loc is the mask

**Comment by Stick-To** [comment]
I think the The location not in groundtruth boxes should be zero, because the conf = class_conf * centerness

**Comment by Stick-To** [comment]
YES, the bceloss is fixed
